Description: Document how quilt searches for its 'patches' directory
 Slight addition to the man page, along with another minor point.
Author: Martin Quinson <mquinson@debian.org>
Last-Update: 2012-12-27
Bug-Debian: http://bugs.debian.org/549482
Upstream-status: committed upstream
 
Index: quilt.git/doc/quilt.1.in
===================================================================
--- quilt.git.orig/doc/quilt.1.in	2012-12-25 14:18:07.446999880 +0100
+++ quilt.git/doc/quilt.1.in	2012-12-27 14:20:59.292052870 +0100
@@ -17,7 +17,7 @@
 With quilt, all work occurs within a single directory tree. Commands can be
 be invoked from anywhere within the source tree. They are of the form
 .B quilt cmd
-similar to CVS commands. They can be abbreviated as long as the specified
+similar to CVS, svn or git commands. They can be abbreviated as long as the specified
 part of the command is unique. All commands print some help text with
 .B quilt cmd -h.
 
@@ -39,10 +39,14 @@
 .I patches
 sub-directory of the source tree (see EXAMPLE OF WORKING TREE below). The 
 .I QUILT_PATCHES
-environment variable can be used to override this location. The
+environment variable can be used to override this location. When not
+found in the current directory, that subdirectory is searched
+recursively in the parent directories (this is similar to the way
+.I git
+searches for its configuration files). The
 .I patches
 directory may contain sub-directories. It may also be a symbolic link
-instead of a directory. 
+instead of a directory.
 
 A file called
 .I series
